src/gui/folderwatcher.cpp

```diff
@@ -149,6 +152,19 @@ void FolderWatcher::changeDetected(const QString &path)
     changeDetected(paths);
 }
 
+void FolderWatcher::notifyChangedPaths()
+{
+    _syncInterval.stop();
+    _lastPaths.clear();
+    if (!_changedPaths.isEmpty()) {
```

Comment:
when can we end up in this method with an empty `_changedPaths` ?

Reply:
Good point, that shouldn't happen.